src/main/resources/com/google/jenkins/plugins/computeengine/InstanceConfiguration/config.jelly

```diff
@@ -89,8 +89,12 @@
                 <f:entry field="machineType" title="${%Machine Type}">
                     <f:select/>
                 </f:entry>
-                <f:entry field="preemptible" title="${%Preemptible?}">
-                    <f:checkbox/>
+                <f:descriptorRadioList descriptors="${descriptor.provisioningTypes}"
```

Comment:
will change to [dropdownDescriptorSelector](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/dropdownDescriptorSelector.jelly); so that jcasc and help text can be supported..

Reply:
implemented. resolving.